assets/src/components/drawer.tsx

```diff
@@ -38,7 +38,7 @@ interface DrawerWithStateProps {
 export const DrawerWithState = (
   props: PropsWithChildren<DrawerWithStateProps>
 ) => {
-  const [drawerOpen, setDrawerOpen] = useState(props.startOpen)
+  const [drawerOpen, setDrawerOpen] = useState(props.startOpen || false)
```

Comment:
Huh, sorry I missed this in my original commit! I had thought I had done this already, but apparently not.